mm/dynamic/c/Magazine.c

```
@@ -1,12 +1,53 @@
+    uint64_t free_slow_paths[NUM_SIZE_CLASSES];
+    uint64_t magazine_swaps[NUM_SIZE_CLASSES];
+    uint64_t slabs_allocated[NUM_SIZE_CLASSES];
+} HeapStatsPerCpu;
+
+static __attribute__((aligned(64))) HeapStatsPerCpu heap_stats_per_cpu[MAX_CPU_CORES];
+
+// Runtime knobs
+static volatile int g_validation_level = KHEAP_VALIDATION_NONE; // 0=none,1=basic,2=full
+static volatile int g_stats_enabled = 1; // can be toggled by perf mode
+
+static inline void StatsAdd(uint32_t cpu, int sc, volatile uint64_t* ctr) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'StatsAdd' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)